config/capsule-answers.yaml

```diff
@@ -3,3 +3,4 @@ certs:
   generate: false
   deploy: true
 capsule: true
+"foreman_proxy::plugin::remote_execution::ssh": false
```

Comment:
Sorry to nitpick, just not sure if it makes a difference: here you have it in quotes, while in katello-answers you don't.